MediaManager/Plugin.MediaManager.Abstractions/IMediaManager.cs

```diff
@@ -20,6 +20,11 @@ namespace Plugin.MediaManager.Abstractions
     public interface IMediaManager
     {
         /// <summary>
+        /// Optional Queue to play media in sequences
+        /// </summary>
+        IMediaQueue Queue { get; set; }

```

Comment:
Nope, because we might want access to the other parts of the Queue from the player.